src/live_restore/live_restore_fs.c

```diff
@@ -1373,27 +1356,29 @@ __wt_live_restore_clean_metadata_string(WT_SESSION_IMPL *session, char *value)
 
         /*
          * Live restore uses -1 in the nbits field to indicate the file has been fully migrated.
-         * This value should be updated to 0 when live restore moves past the background migration
-         * phase, but we can only do so if the file is open when we force a checkpoint during clean
-         * up, or if its btree is dirtied after live restore completes but before restarting in
-         * non-live restore mode. If neither of these events take place the metadata won't be
-         * updated on disk and the -1 value persists. A future live restore using this file as a
-         * source will see this value and incorrectly assume the file has already been migrated. To
-         * prevent this manually overwrite the -1 with the correct value 0.
+         * However if this value is copied into a backup future live restores using this backup as a
+         * source will see the nbits=-1 value and assume a file that still needs migrating has
+         * already been migrated.
```

Comment:
```suggestion
         * However, if this value is copied into a backup, future live restores using this backup as a
         * source will see the nbits=-1 value and assume a file that still needs migrating has
         * already been migrated. Set it to 0 now to indicate it is yet to be migrated.
```
I would also move this down to the line that checks for -1

Reply:
This and the other comments fixed in 21a413a 👍